h2o-core/src/main/java/water/H2O.java

```diff
@@ -618,8 +621,11 @@ else if(s.matches(("client_disconnect_timeout"))){
         }
       else if(s.matches("useUDP")) {
           Log.warn("Support for UDP communication was removed from H2O, using TCP.");
-      }
-      else {
+      } else if (s.matches("features")) {
```

Comment:
An alternative is to have a simple disable_beta and disable_experimental flags. I would not document this option in the usage section (that is just for the commonly used ones).

Reply:
I agree, more selective. Will do it this way. In case of `expert-level` flags, having two instead of one is probably ok.

Reply:
We should probably be explicit regarding the scope of those flags as well:
- do we want to have a generic flag to facilitate our release process and push experimental features earlier, sort of canary version of H2O-3 that would not include only experimental algos but also new APIs or other features like target encoding currently? Also have this in sync with schema versions (currently AutoML is still using schema 99, which makes it kind of 'experimental'...).
In which case, I'd prefer the generic "features" enum valued-flag, but this would also require more thinking I believe, because of the potential scope of such flag... and our internal definition of experimental, beta, stable. Do we really need 'beta' level for example?
- or is it just a convenient flag for algos? 
In which case the flag should be more specific, and still in this case... do we really need 'beta'?

Reply:
given the lines below, it looks like this is targeted specifically for algos, so it should appear in the flag name at least